src/redux/greetings/greetingSlice.js

```diff
@@ -0,0 +1,31 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+export const fetchGreetings = createAsyncThunk('greetings/fetch', async () => {
+  const response = await fetch('http://localhost:5000/api/v1/greetings');
+  const data = await response.json();
+  const { greeting } = data;
+  return greeting;
+});
+
+const greetingSlice = createSlice({
+  name: 'greeting',
+  initialState: { loading: false, greeting: '', error: '' },
+  reducers: {},
+  extraReducers: (builder) => {
+    builder
+      // eslint-disable-next-line max-len
```

Comment:
**OPTIONAL** - Please remove every disabled linter, remember linters help us to write code that is easy to read and maintain. Please fix this, I am only making this optional because it is the only change required.

Reply:
Okay, thanks man. I will do just that